### PR TITLE
test: example of user API from issue 156 was anonymized

### PR DESCRIPTION
**Pull request by Tenischev**
**Description**

Example of user API from issue 156 was anonymized with accordance of request from email.

### CODE REVIEW

**Comment by Tenischev** [comment]
@fmvilas please check

**Review by fmvilas — APPROVED** [review]
LGTM 👍 Once it's merged, I'll ping them again to see if that's enough from their side. Thanks mate!

**Pull request merged**
